cypress/mocks/handlers.ts

```diff
@@ -0,0 +1,128 @@
+import { utcToISTDate } from '@/lib/time-picker-utils';
+import { Platform, STATUS } from '@/types';
+import { Session } from '@/types/api.types';
+import { bypass, delay, http, HttpResponse } from 'msw';
+import initialSessionsData from '../fixtures/initial_sessions.json';
+import { MockLiveData, MockQuizData } from './mockdata';
```

Comment:
can you call them PatchLiveData and PatchQuizData instead? because i think thats what you are using it for?

Reply:
Done